README.md

```diff
@@ -82,6 +86,33 @@ The [contribution guide](CONTRIBUTING.md) describes how to use the `create-rule`
 
 For more advanced command line interface (CLI) usage, refer to the [CLI guide](CLI.md).
 
+## Red Team Automation
+
+The repo comes with some red team automation([RTA](./rta)) python scripts that runs on Windows, Mac OS, and *nix. 
```

Comment:
```suggestion
The repo comes with some red team automation ([RTA](./rta)) python scripts that runs on Windows, Mac OS, and *nix. 
```

Reply:
also i think we could add a brief sentence to explain what RTA exists and why. i think we'll find a lot of users that are unfamiliar with RTA